### DISCUSSION TITLE
when will 5.0 release? I urgently need the bind (key) method...

### DISCUSSION THREAD

**Discussion by Marveee** [TomSchimansky/CustomTkinter#604] [Q&A]
any news?

**Comment by gcailly** [comment]
The "develop" branch is actively developed, with bugfixes lately (4 days ago). Thanks to Tom for all this work !

**Comment by TomSchimansky** [comment]
Version 5.0.0 release isn't out yet because I want to integrate some small changes that brake backwards compatibility, and I want them to integrate in this major version update.
